assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/presentation/member/MemberController.java

```diff
@@ -0,0 +1,45 @@
+package com.splab.invitation.presentation.member;
+
+import com.splab.invitation.domain.member.service.EmailService;
+import com.splab.invitation.domain.member.service.MemberService;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.presentation.common.ApiResponse;
+import jakarta.validation.Valid;
+import java.security.NoSuchAlgorithmException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/members")
+public class MemberController implements MemberApiPresentation {
+
+  private final MemberService memberService;
+  private final EmailService emailService;
+
+  @PostMapping(value = "/invite")
```

Comment:
[구글 API 가이드: 커스터메서드](https://cloud.google.com/apis/design/custom_methods?hl=ko) 를 활용하면 동사를 우아하게 표현할 수 있을 것 같아요.
```suggestion
  @PostMapping(value = "/invitation:send")
```